app/views/users/new.html.erb

```diff
@@ -0,0 +1,31 @@
+<%= form_for @user do |f| %>
+  <fieldset>
+    <legend>Create an Account</legend>
+    <div>
+      <%= f.label :name %>
+      <%= f.text_field :name %>
+    </div>
+
+    <div>
+      <%= f.label :password %>
+      <%= f.password_field :password %>
+    </div>
+
+    <div>
+      <%= f.label :password_confirmation %>
+      <%= f.password_field :password_confirmation %>
+    </div>
+
+    <div>
+      <%= f.submit %>
+    </div>
+  </fieldset>
+<% end %>
+
+<% if @user.errors.any? %>
+  <div class="error_report">
+    <% @user.errors.full_messages.each do |msg| %>
+      <%= msg %>
+    <% end %>
+  </div>
+<% end %>
```

Comment:
You could probably pull this errors block out into a partial - you use it in a number of places.